### PR TITLE
Add new data type example and pom.xml

### PR DESCRIPTION
**Pull request by Cpaulyz**
Add new data type example and pom.xml

### CODE REVIEW

File: java/tsfile/src/main/java/org/apache/tsfile/file/metadata/statistics/BlobStatistics.java [hunk 1]
```diff
@@ -58,7 +58,10 @@ int serializeStats(OutputStream outputStream) throws IOException {
   }
 
   public void updateStats(Binary value) {
-    // do nothing
+    //    if (isEmpty) {
+    //      System.out.println("set isEmpty to false");
+    isEmpty = false;
+    //    }
```
Comment (JackieTien97) [inline java/tsfile/src/main/java/org/apache/tsfile/file/metadata/statistics/BlobStatistics.java]:
```suggestion
    isEmpty = false;
```

File: java/tsfile/src/main/java/org/apache/tsfile/write/record/datapoint/BinaryDataPoint.java [hunk 2]
```diff
@@ -30,14 +30,14 @@
  *
  * @see DataPoint DataPoint
  */
-public class StringDataPoint extends DataPoint {
+public class BinaryDataPoint extends DataPoint {
```
Comment (JackieTien97) [inline java/tsfile/src/main/java/org/apache/tsfile/write/record/datapoint/BinaryDataPoint.java]: keep it as `StringDataPoint` to keep compatibility

File: java/tsfile/pom.xml [hunk 3]
```diff
@@ -35,6 +35,11 @@
         <tsfile.ut.skip>${tsfile.test.skip}</tsfile.ut.skip>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>1.3.13</version>
+        </dependency>
```
Comment (JackieTien97) [inline java/tsfile/pom.xml]: did tsfile module need this dependency?

**Review by JackieTien97 — CHANGES_REQUESTED** [review]

File: java/tsfile/src/main/java/org/apache/tsfile/file/metadata/statistics/BlobStatistics.java (hunk 1 above)
Comment (Cpaulyz) [reply java/tsfile/src/main/java/org/apache/tsfile/file/metadata/statistics/BlobStatistics.java]: done

File: java/tsfile/src/main/java/org/apache/tsfile/write/record/datapoint/BinaryDataPoint.java (hunk 2 above)
Comment (Cpaulyz) [reply java/tsfile/src/main/java/org/apache/tsfile/write/record/datapoint/BinaryDataPoint.java]: done

File: java/tsfile/pom.xml (hunk 3 above)
Comment (Cpaulyz) [reply java/tsfile/pom.xml]: I'm not sure. So I revert this modification.

**Review by JackieTien97 — APPROVED** [review]

**Pull request merged**